README.md

```diff
@@ -51,11 +52,14 @@ set of jobs and report their result like in the following screenshot:
 
 #### Using a custom Docker image
 
-The [.circleci/config.yml](dist/.circleci/config.yml) file uses a
-[custom Docker image](https://hub.docker.com/r/juampynr/drupal8ci/) that, although
-generic for Drupal 8 projects, may not fit yours. If this is the case, have a look at it's
+The [CircleCI configuration file](dist/.circleci/config.yml) uses a
+[custom Docker image](https://hub.docker.com/r/juampynr/drupal8ci/) that extends from
+the [official Drupal image](https://hub.docker.com/_/drupal/) and it is [hosted at
+Docker Hub](https://hub.docker.com/r/juampynr/drupal8ci/). If this image
+does not fit your project's architecture, then have a look at its
```

Comment:
Is this still accurate? It looks like it's using the official Drupal image directly.

Reply:
We are now [extending the drupal image](https://hub.docker.com/r/juampynr/drupal8ci/~/dockerfile/) by adding a few libraries on top so we can do testing. Is the text confusing? If so, can you help me to adjust it?